include/extractor/class_data.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef OSRM_EXTRACTOR_CLASSES_DATA_HPP
+#define OSRM_EXTRACTOR_CLASSES_DATA_HPP
+
+#include "util/bit_range.hpp"
+
+#include <cstdint>
+
+namespace osrm
+{
+namespace extractor
+{
+
+using ClassData = std::uint8_t;
+static const std::uint8_t MAX_CLASS_INDEX = 8 - 1;
```

Comment:
We would throw in the extractor callbacks when we detect more then 8 flags. There is a changelog entry for this.